restful.module

```diff
@@ -23,6 +22,7 @@ function restful_ctools_plugin_directory($module, $plugin) {
   if ($module == 'restful') {
     return 'plugins/' . $plugin;
   }
+  return NULL;
```

Comment:
Yes.

At this point everything is converted, but the cleanup work has not been done yet.